src/components/ChartContainer/D3Container.js

```diff
@@ -78,6 +80,23 @@ function D3Container({ dashboardState, dashboardActions, store }) {
         )),
       )
     }
+    if (dates.startDate || dates.endDate) {
```

Comment:
I'm getting an issue when using the other filters that `dates` is undefined.

`if (dates && (dates.startDate || dates.endDate))` only half fixes the problem though. After setting a date range other filters the date range keeps reseting.

Filtering by only dates though, perfect.